examples/ffi/exporter.cpp

```diff
@@ -139,9 +139,13 @@ int main(int argc, char *argv[]) {
   ddog_CharSlice internal_metadata_example = DDOG_CHARSLICE_C(
       "{\"no_signals_workaround_enabled\": \"true\", \"execution_trace_enabled\": \"false\"}");
 
+  ddog_CharSlice system_info_example = DDOG_CHARSLICE_C(
+      "{\"application\": {\"start-time\": \"2023-09-08 13:45:29.415742 UTC\"}, \"platform\": {\"kernel\": \"Darwin Kernel 22.5.0\"}}");
```

Comment:
I noticed that in the RFC, we're using `start_time` and not `start-time`. It may be worth dropping some guidance in the comments for these things?

(Also nitpick: should we use ISO8601 strings for dates?)